packages/pilot/src/components/BalanceTotalDisplay/index.js

```diff
@@ -34,16 +34,17 @@ const BalanceTotalDisplay = ({
         <div className={style.detail}>
           {detail}
         </div>
-        {
-          action &&
-          <Button
-            disabled={disabled}
-            fill="gradient"
-            onClick={action.onClick}
-            size="default"
-          >
-            {action.title}
-          </Button>
+        { action
```

Comment:
aqui talvez seja interessante manter o `&&` na mesma linha do condicional. Porem é oponião pessoal, podemos debater com mais pessoas qual faz mais sentido pro projeto
```suggestion
{ action && (
          <Button
            disabled={disabled}
            fill="gradient"
            onClick={action.onClick}
            size="default"
          >
            {action.title}
          </Button>
        )}
```